source/jsonnet/england-wales/household/blocks/who-lives-here/any_more_people_living_here.jsonnet

```diff
@@ -230,7 +230,7 @@ local editQuestion(questionTitle) = {
       warning: 'All of the information entered about this person will be deleted',
       answers: [
         {
-          id: 'anyone-else-temp-away-remove-confirmation',
+          id: 'any-more-people-living-here-remove-confirmation',
```

Comment:
If you changed it to `any-more-people-living-here-remove-person` then the other ones should be `any-more-people-living-here-remove-person-question` and `any-more-people-living-here-remove-person-confirmation`.